Foundation/NSDecimalNumber.swift

```diff
@@ -9,10 +9,10 @@
 
 
 /***************	Exceptions		***********/
-public let NSDecimalNumberExactnessException: String = "" // NSUnimplemented
-public let NSDecimalNumberOverflowException: String = "" // NSUnimplemented
-public let NSDecimalNumberUnderflowException: String = "" // NSUnimplemented
-public let NSDecimalNumberDivideByZeroException: String = "" // NSUnimplemented
+public let NSDecimalNumberExactnessException: String = "NSDecimalNumberExactnessException"
+public let NSDecimalNumberOverflowException: String = "NSDecimalNumberOverflowException"
+public let NSDecimalNumberUnderflowException: String = "NSDecimalNumberUnderflowException"
+public let NSDecimalNumberDivideByZeroException: String = "NSDecimalNumberDivideByZeroException"
```

Comment:
This and the one below `NSRunLoop` one are from previously a merged commit (cdde78cb7562ab927feb630a7cf5405ff30bc848) from different contributor. There's one other  commit from someone else that got attached to this when I pushed, but that has since been merged as well (65cdfacd1a4ca55874df563364cf146062d02712). Should I just close this and resubmit so it's not as messy?